### PR TITLE
Add schnorr-signatures-jimmy-song-tabconf-2021 to tabconf/2021

### PR DESCRIPTION
**Pull request by Shourya742**
This PR adds [Schnorr Signatures - Jimmy Song - TABConf 2021](https://www.youtube.com/watch?v=Ubp29_DJ5rM) transcript to the tabconf/2021 directory.

### CODE REVIEW

File: tabconf/2021/schnorr-signatures-jimmy-song-tabconf-2021.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+## Q&A
```
Comment (adamjonas) [inline tabconf/2021/schnorr-signatures-jimmy-song-tabconf-2021.md:71]: This section needs to be in Q and A format rather than one long paragraph

**Review by adamjonas — CHANGES_REQUESTED** [review]
@Shourya742 This looks really good. Just need to format that Q and A section and then I think we can merge. Well done!

File: tabconf/2021/schnorr-signatures-jimmy-song-tabconf-2021.md (hunk 1 above)
Comment (Shourya742) [reply tabconf/2021/schnorr-signatures-jimmy-song-tabconf-2021.md:71]: Cool!!! On it!!

**Comment by kouloumos** [comment]
@Shourya742 this indeed looks really good.
My only comment is regarding the math equations. This talk includes a lot of math, which for the audience is presented on the slides, but when reading, the word-by-word explanation isn't the best. What do you think about writing them in code blocks like: `eG = P` to make them more readable? Maybe we should also add that to the reviewing guidelines, for future reviewers. 
I've pushed a commit that applies what I said to the first paragraph.

**Comment by Shourya742** [comment]
@kouloumos I do get your point... While working on this I did felt formatting the math equation would be nice... but I didn't see any example of this kind in rest of the merged PR. So, thought of segregating based on micro topics and making atleast variable's notation related to math equation correct.

**Comment by kouloumos** [comment]
>  but I didn't see any example of this kind in rest of the merged PR.

You are right, this is the first time we have equations in this stage of the project, but transcripts with equations do exist in the repo. You can see some examples [here](https://btctranscripts.com/advancing-bitcoin/2020/2020-02-06-antoine-riard-taproot-lightning/), [here](https://btctranscripts.com/iacr/2021-08-16-jonas-nick-musig2/) and [here](https://btctranscripts.com/london-bitcoin-devs/2020-06-17-tim-ruffing-schnorr-multisig/).

**Comment by Shourya742** [comment]
@kouloumos I have added the asked changes. Kindly review and suggest changes. Thank!!

**Comment by kouloumos** [comment]
Great work! I've pushed a fixup regarding the capitalization of the headings and the name of the file and is now ready to merge!

Thank you @Shourya742!
If you want to have another - more extended look - into Schnorr signatures, "How to Design Schnorr Signatures" by Adam Gibson _(bitcoinology)_ is currently up-for-grabs as a next transcript to review.

If you instead want to explore further the implication of Schnorr signatures and taproot, "Lightning on Taproot" by Arik Sosman _(bitcoinplusplus/layer-2)_  could be an interesting watch/listen/read.

**Pull request merged**